src/components/MainSection/MainSection.tsx

```diff
@@ -1,13 +1,53 @@
-import React, { PureComponent } from "react";
+import React, { PureComponent, FunctionComponent } from "react";
+import PropTypes from 'prop-types';
 import { WithStyles } from "@material-ui/core/styles/withStyles";
 import styles from "./styles";
+import Box from '@material-ui/core/Box';
+import { Container, Tabs, Tab, Typography } from "@material-ui/core";
+import MainSectionState from "./MainSectionState";
 
-export class MainSection extends PureComponent<WithStyles<typeof styles>> {
+const TabContainer: FunctionComponent = (props) => (
+	<Typography component="div" style={{ padding: "1.5rem 0" }}>
+		{props.children}
+  	</Typography>
+);
+
+TabContainer.propTypes = {
```

Comment:
we don't use `propTypes` for typescript implementation of react we use interfaces.
`propTypes` ia a run time check of types but typescript is compile time which is better
check here
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#basic-prop-types-examples